src/backend/marsha/core/tests/test_api_playlist_portability.py

```diff
@@ -0,0 +1,401 @@
+        self.assertQuerysetEqual(video.playlist.portable_to.all(), [])
+
+    def test_no_portability_portable_to_self_and_new(self):
+        """
+        Update playlist portability.
+
+        Portability should be added for a playlist without existing portability
+        when portable_to param contains self playlist id and a playlist id.
+        A 400 error should be raised when self playlist id is sent.
+        """
+        video = factories.VideoFactory()
+        new_playlist = factories.PlaylistFactory()
+
+        response = self._patch_video(
+            video, {"portable_to": [str(video.playlist.id), str(new_playlist.id)]}
+        )
+
+        self.assertEqual(response.status_code, 400)
+        self.assertQuerysetEqual(video.playlist.portable_to.all(), [])
+
+    def test_no_portability_portable_to_self_and_unknown(self):
+        """
+        Update playlist portability.
+
+        No portability should be added for a playlist without existing portability
+        when portable_to param contains self playlist id and an unknown playlist id.
+        A 400 error should be raised when self or unknown playlist id is sent.
+        """
+        video = factories.VideoFactory()
+
+        response = self._patch_video(
+            video, {"portable_to": [str(video.playlist.id), "unknown_id"]}
+        )
+
+        self.assertEqual(response.status_code, 400)
+        self.assertQuerysetEqual(video.playlist.portable_to.all(), [])
+
+    def test_no_portability_portable_to_self_new_and_unknown(self):
+        """
+        Update playlist portability.
+
+        No portability should be added for a playlist without existing portability
+        when portable_to param contains self playlist id and an unknown playlist id.
+        A 400 error should be raised when self or unknown playlist id is sent.
+        """
+        video = factories.VideoFactory()
+        new_playlist = factories.PlaylistFactory()
+
+        response = self._patch_video(
+            video,
+            {
+                "portable_to": [
+                    str(video.playlist.id),
+                    str(new_playlist.id),
+                    "unknown_id",
+                ]
+            },
+        )
+
+        self.assertEqual(response.status_code, 400)
+        self.assertQuerysetEqual(video.playlist.portable_to.all(), [])
+
+    def test_has_portability_no_portable_to(self):
+        """
+        Update playlist portability.
+
+        No portability should be added nor removed for a playlist with an existing portability
+        when no portable_to param is sent.
+        """
+        video = factories.VideoFactory()
+        ported_to_playlist = factories.PlaylistFactory()
+        video.playlist.portable_to.add(ported_to_playlist)
+
+        response = self._patch_video(video, {})
+
+        self.assertEqual(response.status_code, 200)
+        self.assertQuerysetEqual(video.playlist.portable_to.all(), [ported_to_playlist])
```

Comment:
Can you also assert the number of database queries expected. To be sure no `update` has been made here.